### DISCUSSION TITLE
cant see my funds

### DISCUSSION THREAD

**Discussion by twisterdw** [ethereum-optimism/developers#587] [Feedback]
### Feedback

in 2021 i bridged eth from OP to ETH mainnet.so these funds was lost i cant see it no relayer.6 months ago they was visible but not claimeble now these funds are lost.can somebody help?here is bridge TX

0x0f2f70ce16c2739a1d9e1220e075c3d2b4214e105db0390e3afe4f7370af0109

**Comment by imtipi** [comment]
can't find this tx,can you provide your address?

**Comment by twisterdw** [comment]
@imtipi sure 
0x12235b260b113Ebef4A77b27bdc1C8da3E00109D

**Reply by imtipi** [reply]
can find a withdrawal tx at 2021-08-17 08:25,can you create a ticket at discord so that we can have better communication(between us and team)?

**Reply by twisterdw** [reply]
@imtipi already created no response here is ticket link : https://discord.com/channels/667044843901681675/1283037877671825439

**Reply by imtipi** [reply]
joined,waiting team response
